.github/ISSUE_TEMPLATE.md

```diff
@@ -1,7 +1,7 @@
 <!--
 Please consider the following before submitting an issue:
 
-- Issues are reserved for BUG reports and FEATURE requests, DO NOT create issues for questions or support requests.
+- Please ask questions on StackOverflow. Issues are reserved for BUG reports and FEATURE requests, DO NOT create issues for questions or support requests.
```

Comment:
We could even add a link:

```
- Ask questions or support on StackOverflow: https://stackoverflow.com/questions/tagged/chart.js.
- Issues are reserved for BUG reports and FEATURE requests.
- DO NOT create issues for questions or support requests.
````

I would not repeat `Please` and I would split this line in 3 points